frontend/src/components/opportunity/OpportunityIntro.tsx

```diff
@@ -14,10 +14,9 @@ type Props = {
 const OpportunityIntro = ({ opportunityData }: Props) => {
   const t = useTranslations("OpportunityListing.intro");
 
-  const agencyName =
-    opportunityData.summary.agency_name === ""
-      ? "--"
-      : opportunityData.summary.agency_name;
+  const agencyName = opportunityData.summary.agency_name
```

Comment:
Similar to https://github.com/HHS/simpler-grants-gov/pull/2229 should we change these references to use the root level agency name not the summary?